docs/CONTRIBUTING.md

```diff
@@ -75,70 +88,99 @@ We should create a separate Components repository that implements minimal Codest
+### Read the following if you have not already
+* [Atomic Design](http://bradfrost.com/blog/post/atomic-web-design/)
+* [Atoms](http://bradfrost.com/blog/post/atomic-web-design/#atoms)
+* [Molecules](http://bradfrost.com/blog/post/atomic-web-design/#molecules)
+* [Organisms](http://bradfrost.com/blog/post/atomic-web-design/#organisms)
+* [Templates](http://bradfrost.com/blog/post/atomic-web-design/#templates)
+* [Pages](http://bradfrost.com/blog/post/atomic-web-design/#pages)
+
+### Run Storybook component explorer for development using `start-storybook` npm script
+
+### Run Jest for unit and snapshot tests using `test` or `test:watch` npm script
+
+### Place React UI component under one of the following folders under an appropriate category
+
+#### Atoms `src/components/Atoms`
+Stateless components who do not rely on other (internal components);
+* Button
+* Heading
+* Icon
+* Label
+* Link
+* Input
+* Paragraph
+
+#### Molecules `src/components/Molecules`
+Composition of Atoms;
+* Form Field: using Atoms: Label, Input
+* Search Form: using Atoms: Label, Input
+* Introduction: using Atoms: Heading, Paragraph
+
+#### Organisms `src/components/Organisms`
+Groups of Molecules;
+* Navigation: using Molecules: Buttons, Links
+* Header: using Molecules: Navigation, Search Form, Toggle Button
+* Footer: using Molecules: Heading, Paragraph, Links, Social Icons
+
+#### Templates
+
+#### Pages
 
-* **Run Storybook component explorer for development using `start-storybook` npm script**
+#### Ecosystems
+Composition of one or more Organisms.
+The container component between the (Redux) store / fetching data and the Organism(s) relying on this data.
 
-* **Run Jest for unit and snapshot tests using `test` or `test:watch` npm script**
+#### Environments
 
-* **Place React UI component under one of the following folders under an appropriate category**
-  * Atoms: `src/components/Atoms`
-  * Molecules: `src/components/Molecules`
-  * Organisms: `src/components/Organisms`
-  * Templates: `src/components/Templates`
-  * Pages: `src/components/Pages`
+* Root components, like `<App />`  
 
-* **Build React UI component implementing the following:**
-  * `index.ts`
-    ```typescript
-    import [AwesomeComponent] from './[AwesomeComponent]';
+### Build React UI component implementing the following:
+* `index.ts`
+  ```typescript
+  import [AwesomeComponent] from './[AwesomeComponent]';
 
-    export default [AwesomeComponent];
-    ```
+  export default [AwesomeComponent];
+  ```
 
-  * `[AwesomeComponent].tsx`
-    ```typescript
-    import React from 'react';
-    import classNames from 'classnames/bind';
-    import style from './AwesomeComponent.module.css';
+* `[AwesomeComponent].tsx`
+  ```typescript
+  import React from 'react';
+  import classNames from 'classnames/bind';
+  import style from './AwesomeComponent.module.css';
 
-    const cx = classNames.bind(style);
+  const cx = classNames.bind(style);
```

Comment:
@mvdam and me are curious about this, we are familiar with `classNames` but no with `classNames.bind`

Reply:
Maybe @mts can answer this?